### DISCUSSION TITLE
第6节课作业(6班)

### DISCUSSION THREAD

**Discussion by zhuyr97** [InternLM/Tutorial#298] [作业]
第6节课作业(6班)

**Comment by zhangzefeng92** [comment]
微信名：捂上眼睛看世界
![图片](https://github.com/InternLM/tutorial/assets/58167242/de023f41-34e2-4235-b996-ea8c9a7de5be)

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/b50ffde4-cd45-4172-b0ea-88fcd2ff1990](https://app.yinxiang.com/fx/b50ffde4-cd45-4172-b0ea-88fcd2ff1990)

**Comment by RalphGuo** [comment]
guos-https://gitee.com/RalphGs/InternLM-assignment/tree/master/%E8%AF%BE%E7%A8%8B6
